lib/eex/test/eex_test.exs

```diff
@@ -274,35 +274,157 @@ defmodule EExTest do
 
   describe "raises syntax errors" do
     test "when the token is invalid" do
-      assert_raise EEx.SyntaxError, "nofile:1:12: missing token '%>'", fn ->
+      message = """
+      nofile:1:5: missing token '%>'
+
+        |
+      1 | foo <%= bar
+        |        ^
```

Comment:
I think the `^` is out of place. It should be one character ahead. I think you need to consider `length(marker)` when positioning the arrow.